dev/ci/user-overlays/README.md

```diff
@@ -21,14 +21,11 @@ and the branch name), then a `.sh` extension (`[0-9]{5}-[a-zA-Z0-9-_]+.sh`).
 Example: `13128-SkySkimmer-noinstance.sh` containing
 
 ```
-if [ "$CI_PULL_REQUEST" = "13128" ] || [ "$CI_BRANCH" = "noinstance" ]; then
-
-    overlay elpi https://github.com/SkySkimmer/coq-elpi noinstance
-
-fi
+overlay elpi https://github.com/SkySkimmer/coq-elpi noinstance 13128
 ```
 
-(`CI_PULL_REQUEST` and `CI_BRANCH` are set in [`ci-common.sh`](../ci-common.sh))
+See [`ci-common.sh`](../ci-common.sh) for the detailed documentation of
+the `overlay` function.
```

Comment:
If what is referred from here is just this:

```
# overlay <project> <giturl> <ref> <prnumber> [<prbranch>]
#   creates an overlay for project using a given url and branch which is
#   active for prnumber or prbranch. prbranch defaults to ref.
```

Then I don't think it's very wise to refer to the code instead of updating the (almost complete) doc in this file.